database/seeders/DatabaseSeeder.php

```diff
@@ -30,6 +30,15 @@ public function run(): void
         \App\Models\User::factory()->create([
             'name' => 'Test User',
             'email' => 'test@example.com',
+            'password' => bcrypt('123456'),
         ]);
+
+        // ایجاد کاربر ادمین
+        \App\Models\User::factory()->create([
+            'name' => 'Admin',
+            'email' => 'admin@example.com',
+            'password' => bcrypt('123456'),
```

Comment:
- it is better to use `Hash:make`
```suggestion
            'password' => Hash::make('123456'),
        ]);

        // ایجاد کاربر ادمین
        \App\Models\User::factory()->create([
            'name' => 'Admin',
            'email' => 'admin@example.com',
            'password' => Hash::make('123456'),
```